pic-sure-hpds-ui/src/main/webapp/picsureui/output/dataSelection.js

```diff
@@ -214,9 +214,9 @@ define(["jquery", "common/spinner", "backbone", "handlebars", "text!output/dataS
 				this.$el.html(this.template(this.settings));
 
 				spinner.small(
-					// ontology.tree builds a tree of json objects, and passes it to the innter function which is
+					// tree.getInstance builds a tree of json objects, and passes it to the innter function which is
 					// responsible for rendering the elements.
-					ontology.tree(function(tree){
+					tree.getInstance(function(tree){
```

Comment:
when I see 'getInstance' is suggests a method that returns an instance of a variable or object.  maybe 'initialize' or 'build' would be clearer here.